rules/linux/macos_system_net_disc_firewall_enum.yml

```diff
@@ -0,0 +1,24 @@
+title: System Network Discovery - Firewall Enumeration
+id: 71da9e5a-fb1e-46a8-abc1-28c80173af4c
+status: experimental
+description: Detects enumeration of firewall configuration
+author: remotephone, oscd.community
+date: 2020/10/06
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1016/T1016.md
+logsource:
+    product: macos
+detection:
+    selection:
+        ParentCommandLine|contains:
+            - 'netstat'
+            - 'ifconfig'
+            - 'defaults read /Library/Preferences/com.apple.alf'
+            - 'socketfilterfw'
```

Comment:
could you please change this part to processname/commandline? 

Here is how `netstat` execution looks like in Santa log:

```
 "process": {
      "args": [
        "netstat"
      ],
      "executable": "/usr/sbin/netstat",
    },
```

here is how ifconfig execution looks like:

```
"process": {
      "args": [
        "/sbin/ifconfig"
      ],
      "executable": "/sbin/ifconfig"
    },
```

here is how `defaults read /Library/Preferences/com.apple.alf` execution looks like:

```
"process": {
      "args": [
        "defaults",
        "read",
        "/Library/Preferences/com.apple.alf"
      ],
      "executable": "/usr/bin/defaults"
    },
```

Here is how `/usr/libexec/ApplicationFirewall/socketfilterfw --getglobalstate` execution looks like:

```
"process": {
      "args": [
        "/usr/libexec/ApplicationFirewall/socketfilterfw",
        "--getglobalstate"
      ],
      "executable": "/usr/libexec/ApplicationFirewall/socketfilterfw",
    },
```

So the idea is to split processname and commandline where it is possible. it is cheaper from processing point of view.